ERCS/erc-9999.md

```diff
@@ -0,0 +1,635 @@
+---
+eip: 9999
+title: Readable Typed Signatures for Smart Accounts
+description: Nested typed data structure for readable replay-safe signatures
+author: vectorized (@vectorized), Sihoon Lee (@push0ebp), Francisco Giordano (@frangio), Im Juno (@junomonster), howydev (@howydev), 0xcuriousapple (@0xcuriousapple)
+discussions-to: https://ethereum-magicians.org/t/readable-typed-signatures-for-smart-accounts/20513
```

Comment:
```suggestion
discussions-to: https://ethereum-magicians.org/t/erc-7739-readable-typed-signatures-for-smart-accounts/20513
```

Reply:
The Eth Magicians topic can just include a link to this PR rather than copying the entire contents of the draft ERC.